core/src/test/java/uk/gov/gchq/koryphe/tuple/binaryoperator/TupleAdaptedBinaryOperatorCompositeTest.java

```diff
@@ -15,29 +15,33 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class TupleAdaptedBinaryOperatorCompositeTest
+        extends BinaryOperatorTest<TupleAdaptedBinaryOperatorComposite<Object>> {
 
-public class TupleAdaptedBinaryOperatorCompositeTest extends BinaryOperatorTest<TupleAdaptedBinaryOperatorComposite> {
     @Test
     @Override
+    @SuppressWarnings("unchecked")
     public void shouldJsonSerialiseAndDeserialise() throws IOException {
         // Given
-        TupleAdaptedBinaryOperatorComposite instance = getInstance();
-        String json = "" +
+        final TupleAdaptedBinaryOperatorComposite<Object> instance = getInstance();
+        final String json = "" +
                 "{" +
                     "\"operators\": [" +
                         "{" +
                             "\"selection\": [\"input\", \"anotherInput\"]," +
                             "\"binaryOperator\": {" +
                                 "\"class\": \"uk.gov.gchq.koryphe.impl.binaryoperator.Sum\"" +
-                            "}" +
+                             "}" +
```

Comment:
What's this? It looks to me like the indentation was correct before this change.